operator/src/test/java/oracle/kubernetes/operator/helpers/EventHelperTest.java

```diff
@@ -0,0 +1,408 @@
+import static oracle.kubernetes.operator.helpers.EventHelper.EventItem.DOMAIN_PROCESSING_COMPLETED;
+import static oracle.kubernetes.operator.helpers.EventHelper.EventItem.DOMAIN_PROCESSING_FAILED;
+import static oracle.kubernetes.operator.helpers.EventHelper.EventItem.DOMAIN_PROCESSING_RETRYING;
+import static oracle.kubernetes.operator.helpers.EventHelper.EventItem.DOMAIN_PROCESSING_STARTING;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+public class EventHelperTest {
+  private static final String OPERATOR_POD_NAME = "my-weblogic-operator-1234";
+  private static final String OP_NS = "operator-namespace";
+
+  private final List<Memento> mementos = new ArrayList<>();
+  private final KubernetesTestSupport testSupport = new KubernetesTestSupport();
+  private final DomainProcessorDelegateStub processorDelegate = DomainProcessorDelegateStub.createDelegate(testSupport);
+  private final DomainProcessorImpl processor = new DomainProcessorImpl(processorDelegate);
+  private final Domain domain = DomainProcessorTestSetup.createTestDomain();
+  private final Map<String, Map<String, DomainPresenceInfo>> presenceInfoMap = new HashMap<>();
+  private final DomainPresenceInfo info = new DomainPresenceInfo(domain);
+  private final MakeRightDomainOperation makeRightOperation
+      = processor.createMakeRightOperation(info);
+  private final String jobPodName = LegalNames.toJobIntrospectorName(UID);
+
+  @Before
+  public void setUp() throws Exception {
+    mementos.add(TestUtils.silenceOperatorLogger());
+    mementos.add(testSupport.install());
+    mementos.add(StaticStubSupport.install(DomainProcessorImpl.class, "DOMAINS", presenceInfoMap));
+    mementos.add(TuningParametersStub.install());
+    mementos.add(HelmAccessStub.install());
+
+    testSupport.addToPacket(JOB_POD_NAME, jobPodName);
+    testSupport.addDomainPresenceInfo(info);
+    testSupport.defineResources(domain);
+    DomainProcessorTestSetup.defineRequiredResources(testSupport);
+    HelmAccessStub.defineVariable(OPERATOR_NAMESPACE_ENV, OP_NS);
+    HelmAccessStub.defineVariable(OPERATOR_POD_NAME_ENV, OPERATOR_POD_NAME);
+  }
+
+  @After
+  public void tearDown() {
+    mementos.forEach(Memento::revert);
+  }
+
+  @Test
+  public void whenDomainMakeRightCalled_domainProcessingStartedEventCreated() {
+    makeRightOperation.execute();
+
+    assertThat("Event DOMAIN_PROCESSING_STARTED",
+        containsEvent(getEvents(), DOMAIN_PROCESSING_STARTING_EVENT), is(Boolean.TRUE));
+  }
+
+  @Test
+  public void whenDomainMakeRightCalled_domainProcessingStartedEventCreatedWithExpectedNamespace() {
+    makeRightOperation.execute();
+
+    assertThat("Event DOMAIN_PROCESSING_STARTED message",
+        containsEventWithNamespace(getEvents(),
+            DOMAIN_PROCESSING_STARTING_EVENT),
+        is(Boolean.TRUE));
```

Comment:
you can simply use `is(true)` here. The compiler will do the necessary unboxing.